okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -103,12 +104,47 @@
     assertEquals(200, response.code());
     response.body().close();
 
+    // ordering of response body end currently differs
+    List<String> expectedEvents;
+    if (response.protocol == Protocol.HTTP_2) {
```

Comment:
Should we break this out into distinct test cases? We could disable HTTP/2 on the client in one case, and enforce it on the other.